src/features/components/templates/trade-platform/other-platforms/styles.module.scss

```diff
@@ -0,0 +1,28 @@
+.slider_wrapper {
+    max-width: 96.5rem;
+    margin-inline: auto;
+    position: relative;
+    --swiper-pagination-bottom: 0;
+    --swiper-pagination-color: #ff444f;
```

Comment:
Maybe could use variable **color-red** since we have it in the codebase

Reply:
Thanks

Reply:
Somehow, variable doesn't work here.